roles/confluent.test/molecule/Dockerfile-debian.j2

```diff
@@ -0,0 +1,13 @@
+FROM {{ item.image }}
+
+RUN apt-get update && \
+    apt-get install --yes \
+      apt-transport-https \
+      gnupg \
+      software-properties-common \
+      openjdk-8-jdk \
+      rsync \
+      ca-certificates \
+      openssl \
+      unzip \
+      curl
```

Comment:
Is this dockerfile and the other one missing the confluent packages?

Reply:
Yep, it just installs what is missing to get the scenario working as the images don't contain all required packages.

Reply:
yep i remember now, i think in the 600 branch (or 550) i improved the common role so this is not necessary, but it def is in 540